jsonnet/cluster-monitoring-operator.jsonnet

```diff
@@ -128,5 +128,23 @@ local metrics = import 'telemeter-client/metrics.jsonnet';
       clusterRole.new() +
       clusterRole.mixin.metadata.withName('monitoring-rules-edit') +
       clusterRole.withRules(rules),
+
+    userWorkloadConfigEditRole:
+      local role = k.rbac.v1.role;
+      local policyRule = role.rulesType;
+
+      local configmapRule = policyRule.new() +
+                            policyRule.withApiGroups(['']) +
+                            policyRule.withResources([
+                              'configmaps',
+                            ]) +
+                            policyRule.withVerbs(['*']) +
+                            policyRule.withResourceNames(['user-workload-monitoring-config']);
+
+      role.new() +
+      role.mixin.metadata.withName('user-workload-monitoring-config-edit') +
```

Comment:
nit: we specified `monitoring-workload-config-edit` in https://issues.redhat.com/browse/MON-984.

Reply:
And I asked the comment on that issue why? :) As I am not sure that makes sense, as the role edits user workload no?

Reply:
oh sorry, my jira notification iterations are in the mornings. I remember it was a conscious decision by PM @sichvoge as he does not want to leak the term "user workload monitoring" to end user APIs. Let me comment on the jira.